.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+- name: Ideas & Feature Requests
+  url: https://github.com/TheSuperHackers/GeneralsGamePatch/discussions/categories/ideas
```

Comment:
Is this meant to link to GeneralsGamePatch (the other repo)?

Reply:
No it isn't, thanks for catching that.